### PR TITLE
Adding r-gggenomes

### PR DESCRIPTION
**Pull request by DonatWulf**
@conda-forge/help-r, ready for review!
@conda-forge-admin, please ping team

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by DonatWulf** [comment]
@conda-forge/help-r

**Comment by DonatWulf** [comment]
the staged-recipes (win_64 win) test encounters the following error:
- nothing provides r-base >=4.2,<4.3.0a0 needed by r-ggplot2-3.5.0-r42hc72bb7e_0
How can I fix this?

**Comment by DonatWulf** [comment]
> the staged-recipes (win_64 win) test encounters the following error:
> 
>     * nothing provides r-base >=4.2,<4.3.0a0 needed by r-ggplot2-3.5.0-r42hc72bb7e_0
>       How can I fix this?

Fixed it! All checks have passed now.

File: recipes/r-gggenomes/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    - '*/R.dll'        # [win]
+    - '*/Rblas.dll'    # [win]
+    - '*/Rlapack.dll'  # [win]
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base >=4.2
```
Comment (mfansler) [inline recipes/r-gggenomes/meta.yaml:32]: Please restore to unspecified:

```suggestion
    - r-base
```

File: recipes/r-gggenomes/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    - r-rlang
+    - r-scales
+    - r-snakecase
+    - r-stringr
+    - r-tibble
+    - r-tidyr
+    - r-tidyselect
+    - r-vctrs
+  run:
+    - r-base >=4.2
```
Comment (mfansler) [inline recipes/r-gggenomes/meta.yaml:50]: Please restore to unspecified:

```suggestion
    - r-base
```

File: recipes/r-gggenomes/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+    by 'ggraph' and introduces tracks to bring tidiness to the mess that is genomics
+    data.
+  license_family: MIT
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/MIT'
+    - LICENSE
+
+extra:
+  recipe-maintainers:
+    - DonatWulf
```
Comment (mfansler) [inline recipes/r-gggenomes/meta.yaml:88]: Please retain the CF R team (the linter complaint can be ignored):

```suggestion
    - conda-forge/r
    - DonatWulf
```

**Review by mfansler — CHANGES_REQUESTED** [review]
It is acceptable for `noarch` recipes to fail on **win-64** platform. Essentially, the recipe was fine prior to the adjustments, so I've added comments for where to revert.

In the future, please refrain from requesting review until the CI checks have completed.

**Comment by jakirkham** [comment]
Done in PR: https://github.com/conda-forge/staged-recipes/pull/27385

**Pull request closed without merging**